nautobot/core/templates/inc/footer.html

```diff
@@ -2,53 +2,103 @@
-                    <p class="text-muted">{% now 'Y-m-d H:i:s T' %}</p>
-                </div>
-                <div class="col-xl-4 col-lg-5 col-md-6 col-sm-12 col-xs-12 text-right footer-col-3 noprint">
-                    {% if request.user.is_authenticated  %}
-                    <p class="text-muted footer-links">
-                        <a href="#theme_modal" data-bs-toggle="modal" data-bs-target="#theme_modal" id="btn-theme-modal"><i class="mdi mdi-theme-light-dark text-primary"></i>Theme</a> &middot;
-                        <i class="mdi mdi-book-open-page-variant text-primary"></i>
-                        {% if settings.BRANDING_URLS.docs %}
-                            <a href="{{ settings.BRANDING_URLS.docs }}">Docs</a>
-                        {% else %}
-                            <a href="{% static 'docs/index.html' %}">Docs</a>
-                        {% endif %}
+<footer id="footer">
+    <div class="container-fluid bg-body-secondary text-secondary">
+        <div class="row row-cols-3">
+            <div class="col align-items-center d-flex gap-8 fs-5">
+                {% if request.user.is_authenticated %}
+                    {% if settings.BRANDING_FILEPATHS.logo and settings.BRANDING_POWERED_BY_URL %}
+                        <a
+                            class="align-items-center d-inline-flex gap-4 text-decoration-none"
+                            href="{{ settings.BRANDING_POWERED_BY_URL }}"
+                            target="_blank"
+                        >
+                            <img alt="Nautobot" src="{% static 'img/nautobot_logo.svg' %}" style="height: 1.25rem;" />
+                            Powered
+                        </a>
                         &middot;
-                        <img src="{% static 'img/jinja_logo.svg' %}" style="height:20px"> <a href="{% url 'render_jinja_template' %}">Jinja Renderer</a> &middot;
-                        <i class="mdi mdi-cloud-braces text-primary"></i> <a href="{% url 'api_docs' %}">API</a> &middot;
-                        <i class="mdi mdi-graphql text-primary"></i> <a href="{% url 'graphql' %}">GraphQL</a> &middot;
-                        <i class="mdi mdi-xml text-primary"></i> <a href="{{ settings.BRANDING_URLS.code }}">Code</a> &middot;
-                        <i class="mdi mdi-lifebuoy text-primary"></i> <a href="{{ settings.BRANDING_URLS.help }}">Help</a>
-                    </p>
                     {% endif %}
-                </div>
+                    <span>
+                        {{ settings.HOSTNAME }}
+                        {% if request.user.is_authenticated %}
+                            (v{{ settings.VERSION }})
+                        {% endif %}
+                    </span>
+                {% endif %}
+            </div>
+            <div class="col lh-sm fs-5 py-10 text-center">
+                {% now 'Y-m-d H:i:s T' %}
```

Comment:
Not at all blocking for this PR but we'll want to revisit this as a part of #7018.